buildSrc/src/main/kotlin/io/spine/internal/gradle/Runtime.kt

```diff
@@ -60,23 +60,26 @@ class Cli(private val workingFolder: File) {
         val outWriter = StringWriter()
         val errWriter = StringWriter()
 
-        val process = ProcessBuilder(*command)
-            .directory(workingFolder)
-            .redirectOutput(ProcessBuilder.Redirect.PIPE)
-            .redirectError(ProcessBuilder.Redirect.PIPE)
-            .start()
+        val process = ProcessBuilder(*command).apply {
+            directory(workingFolder)
+            redirectOutput(PIPE)
+            redirectError(PIPE)
+        }.start()
```

Comment:
Seems like there's now more symbols to read. I'd stipulate that method chaining is better than scope functions. At least, there's not another technical "word" that gets in the way of reading the code.